toolkits/springernature/packages/springernature-submission-header/README.md

```diff
@@ -54,9 +54,15 @@ Import the styles into your main stylesheet
 // enhanced.scss
 @import '@springernature/brand-context/springernature/scss/10-settings/colors/default';
 @import '@springernature/springernature-submission-header/scss/10-settings/spacing';
+@import '@springernature/brand-context/default/scss/10-settings/typography.scss';
```

Comment:
Remove the scss extension on typography and arrow for consistency with other imports?